oeq-ts-rest-api/src/index.ts

```diff
@@ -0,0 +1,4 @@
+export * as Auth from './Auth';
+export * as LegacyContent from './LegacyContent';
+export * as Errors from './Errors';
+export * as Settings from './Settings';
```

Comment:
Nice to have, a newline at the end of the file

```suggestion
export * as Settings from './Settings';

```